src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/Scenarios/FunctionProbes/FunctionProbesScenario.cs

```diff
@@ -148,6 +148,40 @@ private static async Task Test_CaptureNativeIntegersAsync(FunctionProbesManager
             }, token);
         }
 
+        private static async Task Test_CaptureValueTypeTypeSpecsAsync(FunctionProbesManager probeManager, PerFunctionProbeProxy probeProxy, CancellationToken token)
+        {
+            MethodInfo method = typeof(StaticTestMethodSignatures).GetMethod(nameof(StaticTestMethodSignatures.ValueType_TypeSpec));
+            (int?, bool?) testTuple = (null, true);
+
+            await RunStaticMethodTestCaseAsync(probeManager, probeProxy, method, new object[]
+            {
+                null,
+                testTuple
+            }, token);
+        }
+
+        private static async Task Test_CaptureGenericsAsync(FunctionProbesManager probeManager, PerFunctionProbeProxy probeProxy, CancellationToken token)
+        {
+            MethodInfo method = Type.GetType($"{nameof(SampleMethods)}.GenericTestMethodSignatures`2").GetMethod("GenericParameters");
+            Assert.NotNull(method);
+
+            List<List<int?>> ints = [[10]];
```

Comment:
Doesn't have to be part of this checkin, but it may be worthwhile to add a broader set of generics in the unit test.